### PR TITLE
✨ added `STORAGE_SECURE` to `StorageAuthSettings`

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Allows to configure the endpoint to use `https` instead of the default `http`.

## Related issue/s

- https://github.com/ITISFoundation/osparc-simcore/pull/5361

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/settings-library/src/settings_library/node_ports.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 class StorageAuthSettings(StorageSettings):
     STORAGE_USERNAME: str | None
     STORAGE_PASSWORD: SecretStr | None
+    STORAGE_SCHEME: str = "http"
```
Comment (sanderegg) [inline packages/settings-library/src/settings_library/node_ports.py:16]: ok can yo uplease check the other ENV and use the same wording? it is already complicated enough.
Either go with:
- STORAGE_URL
- or, STORAGE_SECURE
- or change all of them to SCHEME

**Review by sanderegg — COMMENTED** [review]
Please choose a wording and use it all over the place.

**Review by matusdrobuliak66 — APPROVED** [review]
Thanks :+1:

File: packages/settings-library/src/settings_library/node_ports.py (hunk 1 above)
Comment (pcrespov) [reply packages/settings-library/src/settings_library/node_ports.py:16]: As discussed, let's adapt and go for `{prefix_}_SECURE` by modifying `MixinServiceSettings`

**Review by pcrespov — APPROVED** [review]

**Review by YuryHrytsuk — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
thanks!

**Pull request merged**